utils/utils.js

```diff
@@ -212,6 +213,91 @@ function getRevenue(properties, eventName) {
   return getCurrency(revenue);
 }
 
+/**
+ *
+ *
+ * @param {*} integrationObject
+ */
+function tranformToRudderNames(integrationObject) {
+  Object.keys(integrationObject).forEach(key => {
```

Comment:
should we make this search case insensitive?

Reply:
skipping as discussed